CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ if(CMAKE_BUILD_TYPE MATCHES Debug)
 endif()
 if(CMAKE_BUILD_TYPE MATCHES Release)
   message(STATUS "Configuring OSRM in release mode")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer")
```

Comment:
Please make sure to remove this before merging.

(I usually set  `CXXFLAGS` in the env for flags like needed by perf, then I don't have to modify the build system)